IFAS/DAL/IFDAL.py

```diff
@@ -0,0 +1,26 @@
+from db import db_connection
+from sqlalchemy import text
+
+class ifData:
+    def __init__(self):
+        self.db = db_connection()
+
+    def getIf(self, id: str) -> list:
+        wantedIf = self.db.execute(text("SELECT * FROM ifs WHERE id = :id"),{"id": str(id)}).mappings().all()
```

Comment:
this is a bad practise, tell me why + change it to use baseModel, same for the rest of the file